### DISCUSSION TITLE
brainstorm 🧠

### DISCUSSION THREAD

**Discussion by chickenkatsu1** [chickenkatsu1/parakeet#4] [Ideas]
## 🦜
  drop ideas below!

**Comment by chickenkatsu1** [comment]
infinite scroll

**Comment by acxz** [comment]
- [x] 404 page
- [ ] some vision processing to cut white borders
- [x] dope navbar
 - [ ] navbar doesn't show when scrolling down, shows when u scroll up
- [ ] dependabot
- [ ] commit hooks on eslint
- [ ] typescript
- [ ] scroll with full page display (https://github.com/zwug/react-full-page)
- [ ] namechange/domain ideas? (s - e - e - n - n - i - t)
- [ ] react native
- [ ] host on github pages
